WasatchNET/AgilentSpectrometer.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading;
+using LibUsbDotNet;
+using LibUsbDotNet.Main;
+using SORS3CCA;
+
+namespace WasatchNET
+{
+    public class AgilentSpectrometer : Spectrometer
+    {
+        internal Wrapper wrapper;
```

Comment:
Is this actually a SORS3CCA.Wrapper? I might call it that explicitly, to avoid confusion (many DLLs and namespaces have a Wrapper class).